meta.thrift

```diff
@@ -0,0 +1,4 @@
+service Meta {
+  void health()
```

Comment:
Our health checks currently return 'OK'.

They don't need to do that -- a 200 status code is sufficient, just like an un-errored frame is sufficient here. That "OK" is significant to humans, though (and this endpoint is hit by humans *a lot* when trying to debug issues). If I query a service's health and get `undefined` or `null` back, it's not immediately obvious that that represents a healthy state. `True` is immediately obvious.